iib/workers/tasks/build.py

```diff
@@ -774,14 +771,24 @@ def handle_add_request(
                     ' '.join(excluded_bundles),
                 )
 
-        _opm_index_add(
-            temp_dir,
-            resolved_bundles,
-            prebuild_info['binary_image_resolved'],
-            from_index_resolved,
-            overwrite_from_index_token,
-            (prebuild_info['distribution_scope'] in ['dev', 'stage']),
-        )
+        if is_fbc:
+            opm_registry_add_fbc(
+                base_dir=temp_dir,
+                bundles=resolved_bundles,
+                binary_image=prebuild_info['binary_image_resolved'],
+                from_index=from_index_resolved,
+                overwrite_from_index_token=overwrite_from_index_token,
+                overwrite_csv=(prebuild_info['distribution_scope'] in ['dev', 'stage']),
+            )
+        else:
+            _opm_index_add(
```

Comment:
Is it a correct name of the function? I saw `opm_registry_add` and` _opm_registry_fbc_add` but not `_opm_index_add` function

Reply:
Yes it is, as you can see from git diff. This one is used in current implementation for SQLite based images.